Scripts/script-ExtractDomain.yml

```diff
@@ -75,13 +87,30 @@ script: |-
       markAsIndicator = false;
   }
   for (var i=0; i < uniqueMatches.length; i++) {
-      var domain = extractRootDomain(uniqueMatches[i]);
-      var res = executeCommand("findIndicators", {"query": "type:Domain* and value:\"" + domain + "\""});
-      if (!(res && res[0] && res[0].Contents[0]) && markAsIndicator) {
-          executeCommand("createNewIndicator", {type:'Domain',value:domain, source:'DBot'});
+      var hostname = extractHostname(uniqueMatches[i]);
+      var rootDomain = extractRootDomain(uniqueMatches[i]);
+      var domains = [];
+      domains.push(rootDomain);
+      if (domain != hostname) {
+          domains.push(hostname);
+      }
+      for (var j=0; j < domains.length; j++) {
+          var domain = domains[j];
+          var res = executeCommand("findIndicators", {"query": "type:Domain* and value:\"" + domain + "\""});
+          if (!(res && res[0] && res[0].Contents[0]) && markAsIndicator) {
```

Comment:
maybe add a param to ask if to calculate reputation or not, with default to true ?